source/connect/mysql.txt

```diff
@@ -45,25 +66,64 @@ password after the command has been entered.
 
    mysql '--user=reportsUser?source=admin' --default-auth=mongosql_auth -p
 
-This example assumes that the authentication plugin file
-``mongosql_auth.so`` is located in the default MySQL plugin folder.
-The location of the plugin folder varies by platform, but you can
-locate it by running the following command:
+.. note::
+
+   This example assumes that the authentication plugin file
+   ``mongosql_auth.so`` is located in the default MySQL plugin folder.
+   The location of the plugin folder varies by platform, but you can
```

Comment:
Optional suggestion - I might separate this into two sentences and remove the 'but', so we are left with:

> The location of the plugin folder varies by platform. You can locate it by running the following command: